deployment/helm-k8s/values.yaml

```diff
@@ -5,6 +5,17 @@
 namespace: "siembol"
 environment: "dev"
 
+# -----------------------------------------------------------------------------
+# Choose apps to deploy - default is 'all', you can choose a subset of 
+# apps to deploy by replacing 'all' with the names of the chosen apps, e.g.
+# enabled_apps: 
+# - ui
+# - rest
+# - manager
+# -----------------------------------------------------------------------------
+enabled_apps: 
+- all
```

Comment:
meybe without 'all' it be simpler and more intuitive? For example we do not want in quickstart deployment to have response enabled and you can remove this hidden response flag